packages/svelte-docgen/src/parser/type/constructible.test.ts

```diff
@@ -530,11 +530,11 @@ describe("Constructible", () => {
 			              "alias": "Iterable",
 			              "kind": "interface",
 			              "members": Map {
-			                "__@iterator@88" => {
+			                "__@iterator@19" => {
```

Comment:
. Unrelated to the subject of this issue, but:

No idea why those numbers keep changing. It has to do with internals inside TypeScript Compiler API.

Reply:
**NOTES:**

The numbers change when running a test script from:

- workspace root
- package root

For the sake of CI - use the workspace root.